README.md

```diff
@@ -53,12 +53,10 @@ See the official [React Native website](https://facebook.github.io/react-native/
 
 ## System Requirements
 
-- You can build and deploy React Native Windows apps on the Pro or Enterprise versions of Windows
-	- You cannot run the emulators and some other developer tooling on the "Starter" or "Home" versions of these operating systems
-	- You can run React Native Windows UWP apps only on Windows 10 devices
-       - React Native Windows -[current](https://github.com/Microsoft/react-native-windows/tree/master/current) supports Windows 10 SDK > [14393](https://developer.microsoft.com/en-us/windows/downloads/sdk-archive) and Windows 8.1
-       - React Native Windows -[vnext](https://github.com/Microsoft/react-native-windows/tree/master/vnext) currently supports Windows 10 SDK > [14393](https://developer.microsoft.com/en-us/windows/downloads/sdk-archive)
-    - You can run React Native Windows WPF apps using the [current](https://github.com/Microsoft/react-native-windows/tree/master/current) implementation on Windows 7-10 so long as .NET 4.6 is installed on the end user's machine
+- You can run React Native Windows UWP apps only on Windows 10 devices
+   - React Native Windows -[current](https://github.com/Microsoft/react-native-windows/tree/master/current) supports Windows 10 SDK > [14393](https://developer.microsoft.com/en-us/windows/downloads/sdk-archive) and Windows 8.1
```

Comment:
why Windows 8.1 is here? Will it has conflicts with only Windows 10 devices?

Reply:
The `current` implementation does support Windows 8.1. We also have Issue #2228 tracking work to ensure `vnext` implementation supports Windows 8.1 in the future as well.